azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/controlbar/ControlBarView.kt

```diff
@@ -141,6 +144,6 @@ internal class ControlBarView : LinearLayout {
     }
 
     private fun openAudioDeviceList() {
-        viewModel.displayAudioDeviceSelectionMenu()
+        openAudioDeviceSelectionMenuCallback()
```

Comment:
Can we just get rid of openAudioDeviceList() and have the OnClickListener call openAudioDeviceSelectionMenuCallback() directly?